docs/products/mysql/howto/connect-with-dbeaver.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Connect to Aiven for MySQL® with DBeaver
+sidebar_label: Connect with DBeaver
+---
+
+Use [DBeaver](https://dbeaver.com/) to connect to your Aiven for MySQL® service.
+
+## Prerequisites
+
+- Access to the [Aiven Console](https://console.aiven.io/)
+- At least one running Aiven for MySQL service
+- [DBeaver](https://dbeaver.io/download/) installed on your machine
+
+## Get JDBC URI from Aiven Console
+
+1. Log in to [Aiven Console](https://console.aiven.io/) and navigate to your organization
+   \> project > Aiven for MySQL service.
+1. On the service **Overview** page, select **Quick connect**.
+1. In the **Connect** window
+
+   1. Choose to connect with Java using the **Connect with**
+      dropdown menu.
+   1. Copy the generated JDBC URI.
+   1. Select **Done**.
+
+## Connect to JDBC URI from DBeaver
+
+1. Open DBeaver on your machine, and select **Database** > **New Database Connection**
```

Comment:
```suggestion
1. Open DBeaver and click **Database** > **New Database Connection**
```

Reply:
Here the users actually select options from a number of other possible options available from the top navigation menu. I'd keep "select" in that case. Wdyt?

Reply:
Sure thing, makes sense!